### PR TITLE
Prepare Kyma 1.4.1 patch release

### PR DESCRIPTION
**Pull request by nachtmaar**
**Description**

Changes proposed in this pull request:

- Kyma 1.4.1 release
- Fix initialization of dynamic client #5340
- Use newer console image to fix fetching cmfs via apiserver
- Fix vendored golint 

**Related issue(s)**

https://github.com/kyma-project/kyma/pull/5340

### CODE REVIEW

File: resources/compass/values.yaml [hunk 1]
```diff
@@ -4,23 +4,23 @@ global:
       path: eu.gcr.io/kyma-project/incubator
     connector:
       dir:
-      version: 1.4.0
+      version: 1.4.1
```
Comment (Szymongib) [inline resources/compass/values.yaml]: I am not sure if you can bump images of `compass` as you do not create release branch for it, so the images will not be built.

**Comment by nachtmaar** [comment]
`pre-rel14-kyma-components-function-controller` is optional. I informed @rakesh-garimella about the problem

File: resources/compass/values.yaml (hunk 1 above)
Comment (pkosiec) [reply resources/compass/values.yaml]: The PR is already created on compass repo, so all images have been built.

Comment (nachtmaar) [reply resources/compass/values.yaml]: @Szymongib Please take another look now :)

**Pull request merged**